src/main/java/io/pravega/connectors/flink/sink/PravegaEventWriter.java

```diff
@@ -77,7 +77,7 @@
     private final Stream stream;
 
     // The sink's mode of operation. This is used to provide different guarantees for the written events.
-    private final PravegaWriterMode writerMode;
+    private final DeliveryGuarantee writerMode;
```

Comment:
The variable name can change as well, just call it `deliveryGuarantee`, same to the helper methods